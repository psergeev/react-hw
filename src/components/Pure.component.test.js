import React from 'react';
import renderer from 'react-test-renderer';
import { PureComponent } from './Pure.component';

test('Pure Component works as expected', () => {
    const component = renderer.create(
        <PureComponent/>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});