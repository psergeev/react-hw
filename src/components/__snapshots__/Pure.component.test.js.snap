// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pure Component works as expected 1`] = `
<h2>
  Hello from Pure component
</h2>
`;
